From 04c33c1a952a1c37c51ee8373ebf50bebad3d2c3 Mon Sep 17 00:00:00 2001
From: Mingyen Hung <mingyen.hung@amlogic.com>
Date: Sun, 4 Jul 2021 22:38:39 -0700
Subject: [PATCH] dm-verity: DM Verity support on RDK [4/4]

PD#SWPL-21937

Problem:
Need to support dm-verity on RDK

Solution:
Include image embedded information

Verify:
s905c2(ah212) + RDK firebolt Dunfell

Signed-off-by: Mingyen Hung <mingyen.hung@amlogic.com>
---
 aml-image-packer/s4/aml_upgrade_package.conf     | 4 ++--
 aml-image-packer/s4/aml_upgrade_package_ab.conf  | 6 +++---
 aml-image-packer/sc2/aml_upgrade_package.conf    | 4 ++--
 aml-image-packer/sc2/aml_upgrade_package_ab.conf | 4 ++--
 aml-image-packer/t5d/aml_upgrade_package.conf    | 4 ++--
 aml-image-packer/t5d/aml_upgrade_package_ab.conf | 4 ++--
 6 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/aml-image-packer/s4/aml_upgrade_package.conf b/aml-image-packer/s4/aml_upgrade_package.conf
index 42d38a6..2acd2a7 100644
--- a/aml-image-packer/s4/aml_upgrade_package.conf
+++ b/aml-image-packer/s4/aml_upgrade_package.conf
@@ -18,7 +18,7 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin.signed"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
diff --git a/aml-image-packer/s4/aml_upgrade_package_ab.conf b/aml-image-packer/s4/aml_upgrade_package_ab.conf
index 1c1c3fe..176976a 100644
--- a/aml-image-packer/s4/aml_upgrade_package_ab.conf
+++ b/aml-image-packer/s4/aml_upgrade_package_ab.conf
@@ -19,8 +19,8 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot_a"
 file="boot.img"             main_type="PARTITION"       sub_type="boot_b"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system_a"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system_b"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system_a"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system_b"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin.signed"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
diff --git a/aml-image-packer/sc2/aml_upgrade_package.conf b/aml-image-packer/sc2/aml_upgrade_package.conf
index 42d38a6..2acd2a7 100644
--- a/aml-image-packer/sc2/aml_upgrade_package.conf
+++ b/aml-image-packer/sc2/aml_upgrade_package.conf
@@ -18,7 +18,7 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin.signed"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
diff --git a/aml-image-packer/sc2/aml_upgrade_package_ab.conf b/aml-image-packer/sc2/aml_upgrade_package_ab.conf
index 3659ba9..c74f8f2 100644
--- a/aml-image-packer/sc2/aml_upgrade_package_ab.conf
+++ b/aml-image-packer/sc2/aml_upgrade_package_ab.conf
@@ -18,7 +18,7 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot_a"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system_a"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system_a"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin.signed"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
diff --git a/aml-image-packer/t5d/aml_upgrade_package.conf b/aml-image-packer/t5d/aml_upgrade_package.conf
index 5fed55c..22720f4 100644
--- a/aml-image-packer/t5d/aml_upgrade_package.conf
+++ b/aml-image-packer/t5d/aml_upgrade_package.conf
@@ -18,7 +18,7 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
diff --git a/aml-image-packer/t5d/aml_upgrade_package_ab.conf b/aml-image-packer/t5d/aml_upgrade_package_ab.conf
index 52483bd..ac92a21 100644
--- a/aml-image-packer/t5d/aml_upgrade_package_ab.conf
+++ b/aml-image-packer/t5d/aml_upgrade_package_ab.conf
@@ -18,7 +18,7 @@ file="usb_flow.aml"                    main_type="aml"             sub_type="usb
 
 file="logo.img"         main_type="PARTITION"      sub_type="logo"
 file="boot.img"         main_type="PARTITION"      sub_type="boot_a"
-file="rootfs.ext4.img2simg" main_type="PARTITION"       sub_type="system_a"
-file="vendor.ext4.img2simg" main_type="PARTITION"       sub_type="vendor"
+file="rootfs.ext4.verity.img2simg" main_type="PARTITION"       sub_type="system_a"
+file="vendor.ext4.verity.img2simg" main_type="PARTITION"       sub_type="vendor"
 file="u-boot.bin"   main_type="PARTITION"      sub_type="bootloader"
 file="dtb.img"           main_type="PARTITION"      sub_type="_aml_dtb"
-- 
2.24.1

